metaphlan/3.0.3/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:bionic AS builder
+
+LABEL base.image="ubuntu:bionic"
+LABEL version="1"
```

Comment:
This is an old LABEL that is pretty vague and useless. Please change to this:
`LABEL dockerfile.version="1"`

If we need to adjust the dockerfile (after merging PR into master, building image on dockerhub) then we should increment this to 2. Semantic versioning is overkill here, and I just wanted a way to track dockerfile versions in case we need to tweak them for a specific version of a program. More info here https://github.com/StaPH-B/docker-builds/issues/118

Reply:
Thanks, I updated the dockerfile.version labels in both dockerfiles. 

I also moved the labels down in the Dockerfile, so that the labels are associated with the final docker image that's generated from the multistage build